pkg/logger/logger.go

```diff
@@ -95,14 +94,51 @@ func (l *ZapLogger) FatalWithContext(ctx context.Context, msg string, fields ...
 	l.Logger.Fatal(msg, fields...)
 }
 
-func NewLogger(logFormat, logLevel, logTimestampFormat string) (*ZapLogger, error) {
-	if logLevel == "none" {
+// OptionsLogger Implement options for logger
+type OptionsLogger struct {
+	Format          string
+	Level           string
+	TimestampFormat string
```

Comment:
```suggestion
	format          string
	level           string
	timestampFormat string
```

The member variable should be private.  Changes to these member variables should be done via WithX methods below.

Reply:
Set private